### PR TITLE
Added license_file, license_family, description, doc_url, and dev_url

### LINKED ISSUE #19: Packaging license file
Currently the license file is not packaged. Would be great to have the license file packaged so end users can inspect it after `conda install`ing this package.

ref: https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#license-file

### PR DESCRIPTION
**Pull request by frol**
*Closes #19.*

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -48,7 +48,25 @@ test:
 about:
   home: http://nuitka.net
   license: Apache 2.0
+  license_family: Apache
+  license_file: LICENSE.txt
   summary: Python compiler with full language support and CPython compatibility
+  description: |
+    Nuitka is the Python compiler. It is a seamless replacement or extension
+    to the Python interpreter and compiles every construct that CPython 2.6,
+    2.7, 3.2, 3.3, 3.4, 3.5, and 3.6 have. It then executes uncompiled code,
```
Comment (kayhayen) [inline recipe/meta.yaml]: Reminds me that I have recently removed "3.2" support. I need to find the space where you took that
from and update it.

Comment (frol) [reply recipe/meta.yaml]: I took it from the README.rst overview: https://github.com/kayhayen/Nuitka/blob/develop/README.rst#overview

**Comment by jakirkham** [comment]
Thanks @frol.

Would be good to do a build number bump to get packages built with this license file included.

**Comment by frol** [comment]
@jakirkham Go ahead.

**Pull request merged**